aiida/orm/implementation/querybuilder.py

```diff
@@ -400,7 +398,7 @@ def get_column(self, colname, alias):  # pylint: disable=no-self-use
         try:
             return getattr(alias, colname)
         except AttributeError:
-            raise exceptions.InputValidationError(
+            raise ValueError(
```

Comment:
Should this also `raise ... from ...` ?

Reply:
I don't understand why `pylint` is not complaining about these.